tools/codegen/api/types.py

```diff
@@ -407,11 +407,13 @@ class DispatcherSignature:
     # The schema this signature is derived from
     func: FunctionSchema
 
+    prefix: str = ""
```

Comment:
worth a comment? it opens up a pretty big axis of variation for what the class represents...

Reply:
Definitely fair, I'll add it in